src/inatinqperf/benchmark/configuration.py

```diff
@@ -0,0 +1,62 @@
+"""Modules with classes for loading configurations with Pydantic validation."""
+
+from pathlib import Path
+from typing import Any
+
+from pydantic import BaseModel, PositiveInt
+
+
+class DatasetConfig(BaseModel):
+    """Dataset definition within the main configuration."""
+
+    dataset_id: str
+    # For multiple splits, HuggingFace accepts a single string with splits concatenated with the `+` symbol.
+    splits: str
```

Comment:
Do you want `HuggingFace`'s interface conventions to leak into our abstraction here or would it be better to take a `list[str]` and then convert to `"item1+item2+..."` internally?